nextflow_schema.json

```diff
@@ -1061,7 +1061,8 @@
                     "fa_icon": "far fa-check-circle",
                     "description": "Validation of parameters in lenient more.",
                     "hidden": true,
-                    "help_text": "Allows string values that are parseable as numbers or booleans. For further information see [JSONSchema docs](https://github.com/everit-org/json-schema#lenient-mode)."
+                    "help_text": "Allows string values that are parseable as numbers or booleans. For further information see [JSONSchema docs](https://github.com/everit-org/json-schema#lenient-mode).",
+                    "default": true
```

Comment:
why this change?

Reply:
I used `nf-core schema build`cmd line to update `schema`, was that a mistake?? I read about this in the docs file.[https://training.nextflow.io/nf_develop/1_04_parameters/](url)

Reply:
probably not actually, thanks